### PR TITLE
Assistant: clarify which features are available where

### PR DESCRIPTION
**Pull request by armchairlinguist**
Some sort of edity problem seems to have happened when this page was updated with new info about where different features are active - the new info was added, but the old info wasn't removed, and the formatting became inconsistent.

I'm totally up for further refinements as well, like if we want to switch to callouts.

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [inline docs/semgrep-assistant/overview.md:30]: As far as I know, "self-managed" is GitLab's term for self-hosted. They do also have something called "Dedicated" which I'm not particularly familiar with.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [inline docs/semgrep-assistant/overview.md:117]: I haven't been able to confirm this happens any longer. If we can find confirmation, we can restore this line.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [inline docs/semgrep-assistant/overview.md:78]: I'd like to find a screenshot for this, but I haven't been able to see it happening currently.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [inline docs/semgrep-assistant/overview.md:73]: I don't think this feature is working right now either.

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [reply docs/semgrep-assistant/overview.md:30]: Yes, "self-managed" is the term used by GitLab, so I removed self-hosted for that reason -- do you think that's confusing?

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:52]:
```suggestion
Component tags are available in Semgrep Cloud Platform's **Findings** page.
```

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:58]:
```suggestion
Auto-triage recommendations are available in Semgrep Cloud Platform's **Findings** page when you filter for findings that Assistant suggests should be ignored, and in the [finding's details](/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
```

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:69]:
```suggestion
Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
```

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:71]:
```suggestion
You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant in Semgrep Cloud Platform's **Settings** page.
```

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:73]:
```suggestion
Autofixes are <!-- available in Semgrep Cloud Platform's **Findings** page under **Your code** in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). They are also --> available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
```

I saw that the [fix for this is high priority](https://linear.app/semgrep/issue/AI-505/autofix-not-showing-in-finding-details-for-findings-even-if-they-have), so I suggested we comment it out for the time being

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:82]:
```suggestion
Semgrep sends weekly emails with information on the top three backlog tasks across all findings according to Assistant. Unlike other Assistant features, these suggestions can include information for all Semgrep products that you have enabled. The emails are sent out on Monday to all organization admins.
```

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [reply docs/semgrep-assistant/overview.md:78]: Did you mean a more specific link than something in the Activity section? 

image

Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:78]: It's possible I misunderstood what this is referring to; I was thinking it was part of the info specific to the autofix (which seems not working, as mentioned in my later comment). But if it's only referring to a link to the PR (not the comment specifically - that link will just take you to the PR in general) then this would work.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:69]: Hmm yeah, on its own that comma is definitely wrong, thank you. :) I don't love how the sentence sounds with no commas, though; maybe it needs to be split up properly. What do you think of:

> Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive. Assistant will only suggest an autofix if the rule doesn't have a human-written autofix.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:73]: I'm honestly kinda hoping it'll be fixed before this is merged, but I'll leave this suggestion in case it makes more sense to merge earlier.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:82]: I have the same issue with this sentence sans comma as with the other one, it's just too long to be strung together like that. How about this:

> Semgrep sends weekly emails with information on the Assistant's top three backlog tasks across all findings.

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [reply docs/semgrep-assistant/overview.md:69]: Yes, that looks good!

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [reply docs/semgrep-assistant/overview.md:78]: @jsmoxon can you please clarify what the expected behavior for this is?

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [reply docs/semgrep-assistant/overview.md:82]: That works

File: docs/semgrep-assistant/overview.md
Comment (jsmoxon) [reply docs/semgrep-assistant/overview.md:117]: I don't think we intentionally removed this but I'm not seeing it now either. Fine to remove for now to unblock this.

File: docs/semgrep-assistant/overview.md
Comment (jsmoxon) [reply docs/semgrep-assistant/overview.md:78]: The expected behavior is that where possible, we show an AI autofix code snippet on the Findings details page. We also offer a "copy" snippet button so that users can share it with developers. However, I think this bug is preventing any autofix snippets from being shown: https://linear.app/semgrep/issue/AI-505/autofix-not-showing-in-finding-details-for-findings-even-if-they-have 

@underyx do we include a link to the PR/MR as well?

File: docs/semgrep-assistant/overview.md
Comment (jsmoxon) [reply docs/semgrep-assistant/overview.md:73]: This is blocked on this bug: https://linear.app/semgrep/issue/AI-505/autofix-not-showing-in-finding-details-for-findings-even-if-they-have

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:78]: As Katie brought up, we have a link to the PR/MR in the Activity - the question is more if there ought to be a link that's specifically for going to the comment containing the autofix (this is what I thought it was saying). Or if the wording is ambiguous and it's actually referring to the main link to the PR (which is likely fine, but would benefit from clearer wording so other folks don't get on the track I did).

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:117]: Confirmed it's a bug, hopefully won't take too long to fix, and then we can merge with the line restored.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md]: I changed the wording here a bit so it's clear we link to the PR/MR where the comment exists.

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md]: This is fixed now, yay!

File: docs/semgrep-assistant/overview.md
Comment (khorne3) [inline docs/semgrep-assistant/overview.md:58]:
```suggestion
Auto-triage recommendations are available in SCP's **Findings** page when you filter for findings that Assistant suggests should be ignored and in the [finding's details](/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
```

**Review by khorne3 — APPROVED** [review]

File: docs/semgrep-assistant/overview.md
Comment (armchairlinguist) [reply docs/semgrep-assistant/overview.md:58]: Hmm, this is another one that really doesn't sound right to me without the comma. Technically yes, it's a chain of two basic clauses, but having the conditional "when you filter..." pushes the end of the first main clause a long way from the and, making it hard when you get to the and to figure out where you need to go back and break it.

**Pull request merged**
